### PR TITLE
ss-335: Dropdown placeholder added and updated style

### PR DESCRIPTION
**Pull request by MstyslavDmytryk**

### CODE REVIEW

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/bundles/common/components/dropdown/components/dropdown.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable unicorn/prefer-ternary */
```
Comment (Evcat6) [inline frontend/src/bundles/common/components/dropdown/components/dropdown.tsx:1]: Why we disable eslint there?

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -199,10 +196,7 @@ const Dashboard: React.FC = () => {
...
     );
 
-    const [category, setCategory] = useState<DataType>({
-        value: '',
-        name: 'Find by category',
-    });
+    const [category, setCategory] = useState<DataType>();
 
     const handleDropdownByCategory = useCallback(
         (selectedOption: DataType | null) => {
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:202]: i think that can be helper

**Review by Evcat6 — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/dropdown/components/dropdown.tsx (hunk 1 above)
Comment (MstyslavDmytryk) [reply frontend/src/bundles/common/components/dropdown/components/dropdown.tsx:1]: when I use if operator, eslint suggests ternaty operator
but when I use ternary, there is conflict with eslint and prettier

Comment (Evcat6) [reply frontend/src/bundles/common/components/dropdown/components/dropdown.tsx:1]: ok

**Review by Evcat6 — DISMISSED** [review]

**Review by nikita-remeslov — DISMISSED** [review]

**Comment by Evcat6** [comment]
warning  `'BaseModal' is defined but never used  @typescript-eslint/no-unused-vars`

**Review by Evcat6 — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**